pulpcore/app/models/task.py

```diff
@@ -90,6 +77,7 @@ class Worker(BaseModel):
 
     name = models.TextField(db_index=True, unique=True)
     last_heartbeat = models.DateTimeField(auto_now=True)
+    versions = HStoreField(null=True)
```

Comment:
Shouldn't these always have versions with a default of `{"versions": {}}` based on [this line](https://github.com/pulp/pulpcore/pull/3466/files#diff-7a529f651b0c47d7f7d8b44adaafc9a5fadaa03aa1e455fec23b8ee1c68274e9R49).

Reply:
Maybe it should default to `{}`. So probably:
```suggestion
    versions = HStoreField(default=dict)
```